.github/workflows/update.yml

```diff
@@ -0,0 +1,135 @@
+name: opencti-auto-update
+on:
+  pull_request:
+# schedule:
```

Comment:
Is this useful ?

Reply:
I set it to `pull_request` for testing, it will be set to `schedule` before merging